### ISSUE TITLE
ctypes error when pushing Float Stat

### ISSUE THREAD

**Issue by kennex-software** [philippj/SteamworksPy#40]
Hey Gramps!  

I tried to push a Float Stat to Steam and received this error:
ctypes.ArgumentError: argument 2: <class 'TypeError'>: Don't know how to convert parameter 2

I saw on Stackoverflow, this link: [(https://stackoverflow.com/questions/33710211/how-does-ctypes-know-when-to-throw-argumenterror-but-only-sometimes)]

If I change the SetStat in the UserStats to:

`return self.steam.SetStatFloat(name, c_float(value))`

Then all works well!  Just thought I'd let you know!

**Comment by Gramps** [comment]
Yeah, in my next update I am moving things back to SetStatInt and SetStatFloat; which, obviously, causes less problems.  Right now I'm just bogged down in callbacks and getting my current game ready for Steam Game Festival, but the SteamworksPy update should be in a week or two!  And it's big!

Thanks for the update on that!
